frontend/src/components/design-system/style/BackgroundComponent.vue

```diff
@@ -21,126 +29,46 @@ const currentEmbed = inject(EmbedKey);
     <slot />
   </template>
   <template v-else>
-    <div :class="['background', classes]">
-      <template v-if="props.isImage">
-        <div class="red-spot"></div>
-        <div class="blue-spot"></div>
-        <div class="blue-rounded-spot"></div>
-        <div class="red-rounded-spot"></div>
-        <div class="white-rounded-spot"></div>
-        <div class="gray-rounded-spot"></div>
-      </template>
-      <template v-else>
-        <div class="red-rounded-spot-single"></div>
-        <div class="blue-rounded-spot-single"></div>
+    <div class="has-background">
+      <template v-if="props.hasBlobs">
+        <div class="blobs">
+          <img src="@/assets/background/blue-blob.svg" class="blue-blob"/>
+          <img src="@/assets/background/red-blob.svg" class="red-blob"/>
+        </div>
       </template>
       <slot />
     </div>
   </template>
 </template>
 
 <style scoped lang="scss">
-.background {
+.has-background {
```

Comment:
```suggestion
.background {
```

Máme takovou konvenci, že ta hlavní třída je vždycky stejná jako jméno komponenty.